lib/modulr/client.rb

```diff
@@ -60,7 +60,9 @@ def request(method, path, data = nil, options = {})
       uri = "#{base_url}#{path}"
 
       begin
-        connection.run_request(method, uri, request_options[:body], request_options[:headers])
+        connection.run_request(method, uri, request_options[:body], request_options[:headers]) do |request|
+          request.params.update(options) if options
```

Comment:
This provides a new feature to send query params in the requests using Faraday built-in option.